### ISSUE TITLE
Reproduce MELD F1 score

### ISSUE THREAD

**Issue by so-hyeun** [wxjiao/AGHMN#3]
![reproduced AGHMN](https://user-images.githubusercontent.com/55587907/107207510-b95a7b80-6a43-11eb-8602-df790abfe44d.PNG)

Hello, thanks for doing interesting research.

The picture above is the result of reproducing the MELD dataset performance through UniF-BiAGRU. 
Of the results, is 57.77 the value that should be compared with the paper's F1 score?

**Comment by wxjiao** [comment]
> ![reproduced AGHMN](https://user-images.githubusercontent.com/55587907/107207510-b95a7b80-6a43-11eb-8602-df790abfe44d.PNG)
> 
> Hello, thanks for doing interesting research.
> 
> The picture above is the result of reproducing the MELD dataset performance through UniF-BiAGRU.
> Of the results, is 57.77 the value that should be compared with the paper's F1 score?

Yes, 57.77 corresponds to the F1 score. You can see from the ``EmoTrain.py''. 
![image](https://user-images.githubusercontent.com/31032829/107214907-916c1780-6a45-11eb-8c2a-0c157314c396.png)
